src/components/banner_advanced/html/example1.json

```diff
@@ -70,7 +70,7 @@
           "type": "metadata_field_related_asset",
           "description": "",
           "friendly_name": "Image",
-          "value": "https://via.placeholder.com/782x520",
+          "value": "https://placehold.co/782x520",
```

Comment:
may be best to reference locally since these external sites may not return in time for chromatic